lib/broken_record.rb

```diff
@@ -3,21 +3,52 @@
 require "broken_record/config"
 require "broken_record/scanner"
 require "broken_record/railtie" if defined? Rails::Railtie
+require "broken_record/job_result"
 
 module BrokenRecord
   extend self
 
   def configure
     yield BrokenRecord::Config
 
-    unless BrokenRecord::Config.default_scopes.blank?
+    unless BrokenRecord::Config.default_scopes.nil? || BrokenRecord::Config.default_scopes.count == 0
       ActiveSupport::Deprecation.warn("default_scopes are deprecated and will be removed in the next major version.")
       BrokenRecord::Config.model_includes = BrokenRecord::Config.default_scopes
       BrokenRecord::Config.model_conditions = BrokenRecord::Config.default_scopes
     end
+
+    unless BrokenRecord::Config.aggregator_class.nil?
+      warn_class_string_deprecation('aggregator_class')
+      BrokenRecord::Config.aggregator = get_symbol_from_klass_string(BrokenRecord::Config.aggregator_class)
+    end
+
+    unless BrokenRecord::Config.multi_aggregator_classes.nil? || BrokenRecord::Config.multi_aggregator_classes.count == 0
+      warn_class_string_deprecation('multi_aggregator_classes')
```

Comment:
is the message that get's printed here reference `multi_aggregatores` since you `gsub out `_class`? I'd say you can just write the deprecation out of if you don't want to do the string manipulation

Reply:
Makes sense not much to gain from trying to reuse that